addons/sale/static/src/xml/sale_product_field.xml

```diff
@@ -3,30 +3,26 @@
 <templates xml:space="preserve">
 
     <t t-name="sale.SaleProductField" t-inherit="account.ProductLabelSectionAndNoteField" t-inherit-mode="primary">
-        <!-- Show configuration button for custom lines/products -->
-        <xpath expr="//button[@t-if='hasExternalButton']" position="before">
-            <t t-if="hasConfigurationButton">
+        <xpath expr="//Many2One" position="after">
+            <!-- Show the product quantity next to the product name for combo lines. -->
+            <t t-if="props.readonly and isCombo">
+                <span>
+                    x <t t-out="props.record.data.product_uom_qty"/>
+                </span>
```

Comment:
The display seems a bit broken here, we don't see the quantity.
![image](https://github.com/user-attachments/assets/044528a8-0121-4624-b4a9-5ae052e7f893)

Reply:
fixed